packages/keystone/src/admin-ui/components/index.ts

```diff
@@ -1,8 +1,18 @@
+// FIELD VIEW SPECIFIC COMPONENTS
 export { CellContainer } from './CellContainer';
 export { CellLink } from './CellLink';
+
 export { ErrorBoundary, ErrorContainer } from './Errors';
+
+// ADMIN-UI CUSTOM COMPONENTS
 export { Logo } from './Logo';
-export { Navigation } from './Navigation';
+export { Navigation, NavigationContainer, NavItem, ListNavItems, ListNavItem } from './Navigation';
```

Comment:
When would someone want to use `Navigation`? Also, it is a different thing to the thing that accepts `NavigationProps`, right?

Reply:
Correct, the wrapping `Navigation` component would only be used for custom pages. 
For example if someone were to not want to opt into the entire layout provided by `PageContainer` but did want to have a sidebar nav, they'd leverage that Navigation export